tests/subyt/test_conditional.py

```
@@ -1,96 +1,152 @@
-        D_mtime5 = os.stat(str(SUBYT_TEST_FOLDER / "tmp/data/D.ttl")).st_mtime
-        assert A_mtime4 == A_mtime5  # output file should not have been updated
-        assert D_mtime4 < D_mtime5  # output file should have been updated
-
-        # reset input file
-        df = pd.read_csv(str(SUBYT_TEST_FOLDER / "resources/data.csv"))
-        df.at[0, "value"] = 1
-        df.to_csv(str(SUBYT_TEST_FOLDER / "resources/data.csv"), index=False)
+SUBYT_TEST_FOLDER = Path(__file__).absolute().parent
+
+
+def test_conditional():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_conditional`

Add return type annotation: `None`

(ANN201)